tests/utils/factories.py

```diff
@@ -84,3 +91,19 @@ class Meta:
     name = factory.Faker('company')
     summary = factory.Faker('catch_phrase')
     description = factory.Faker('bs')
+
+
+class _UploaderFactory(factory.Factory):
+    class Meta:
+        model = _Uploader
+
+    @factory.post_generation
+    def assign_values(obj, create, extracted):
```

Comment:
You can just set these values directly like in all of the other factories.  No need for a `post_generation` hook if you are assigning static values like this.

Reply:
I got an error when I tried to set the values directly, I think because they tried to go through the `__init__` method for `_Uploader`, which doesn't take any arguments. 🤷‍♂